src/aind_metadata_service/tars/mapping.py

```diff
@@ -1,11 +1,10 @@
 """Module to map data in TARS to aind_data_schema InjectionMaterial"""
 
+import datetime
```

Comment:
It might be cleaner to do `from datetime import datetime, date` instead of importing the full module